resources/build/build-utils.md

```diff
@@ -144,6 +156,15 @@ another script:
   Options can be used to provide additional data, by including `=<varname>` in
   their definition. Otherwise, they are treated as a boolean.
 
+* **dependencies**: these are other builder scripts which must be configured and
+  built before the actions in this script can continue. Only `configure` and
+  `build` actions are ever passed to dependency scripts; if you are working on
```

Comment:
From a resolved comment near this block:

> > Note that this, in turn, could probably use some formal documentation - these details have been implied until now, but it takes significant inference to come to these conclusions. Note that I only got halfway there with my comment, and I've been using the dependency system for a while.
> >
> > Making this aspect of the design more explicit would definitely help as more people become accustomed to the dependency system.
>
> Happy to hear suggestions on how this could be clearly documented here.

Sorry it took me a while to come up with something; I guess I wasn't quite in the right headspace for it at the time.

```suggestion
  built before the actions in this script can continue. Only `configure` and
  `build` actions are ever passed to dependency scripts.  These actions
  will execute for all targets of the dependency script.  If you are working on
```